src/formats/newick.js

```diff
@@ -13,9 +13,11 @@ import { is_leafnode } from "../nodes";
  * @param {Object} bootstrap_values.
  * @returns {Object} An object with keys ``json`` and ``error``.
  */
-function newick_parser(nwk_str, bootstrap_values) {
-  bootstrap_values = true;
 
+function newick_parser(nwk_str, bootstrap_values, delimiter) {
+  bootstrap_values = true;
+  let left_delimiter = delimiter || '{',
```

Comment:
Is `{` intended to be default? It seems counter-intuitive given line 20.

Reply:
`{` is indeed intended as the default, since I figured we'd want to support the HyPhy format by default. I've only ever come across `{` and `[` for delimiting node annotations (as `(` would clash with delimiting nodes themselves), and lines 19-20 are meant to handle these cases.

Reply:
Cool. A datatype for delimiter that is tuple-like or dictionary-like (e.g. `['{', '}']` or `{ left : '{', right:'}'}` may be more robust, but as you've said, the characters used for annotation are not many.